server/utils/labels.ts

```diff
@@ -0,0 +1,48 @@
+import { IStore } from '../reducers/indexReducer'
+import { state } from '../reducers/store'
+
+export function getChannelLabel(
+    state: IStore,
+    faderIndex: number
+): string | undefined {
+    return state.channels[0].chMixerConnection
+        .map((conn) =>
+            conn.channel.map((ch) => ({
+                assignedFader: ch.assignedFader,
+                label: ch.label,
+            }))
+        )
+        .reduce((a, b) => [...a, ...b], []) // flatten

```

Comment:
can be replaced with: .flat()

Reply:
That's not possible with the current typescript configuration unfortunately.